ereuse_devicehub/resources/models.py

```diff
@@ -34,3 +35,12 @@ def __init__(self, **kwargs) -> None:
         # to be able to use sorted containers
         self.created = kwargs.get('created', datetime.now(timezone.utc))
         super().__init__(**kwargs)
+
+
+def on_update_time(mapper, connection, thing_obj):
+    """ This function update the stamptime of field updated """
+    thing_obj.updated = datetime.now()
+
+def update_timestamp(thing_obj):
```

Comment:
The method name suggest that it will update `thing_obj` `timestamp` but the method only connects a signal listener. Could you update to a more descriptive name?

Reply:
I changed the name for one more descriptive